### PR TITLE
Ephemeral deploy tweaks

### PR DESCRIPTION
**Pull request by coderbydesign**
## Describe your changes

- [Ignore local/auto-generated bonfire config](https://github.com/RedHatInsights/insights-rebac/commit/e3769d86f3ee92d521fea03689b5a86534293aa4)
- [Remove venv in deploy script](https://github.com/RedHatInsights/insights-rebac/commit/ba3d4d9f3fc30da955d2f52d1e10d5e9f2b7becf)
- [Use local config](https://github.com/RedHatInsights/insights-rebac/commit/1374aeba61bcd37dd25b5e02a871559872099f92)
- [Add 'Content-Type: application/json' header to example request](https://github.com/RedHatInsights/insights-rebac/commit/464323889b77dcb85083f868c392c5360cc78f59)

## Checklist

* [ ] Are the agreed upon acceptance criteria fulfilled?

* [ ] Was the 4-eye-principle applied? (async PR review, pairing, ensembling)

* [ ] Do your changes have passing automated tests and sufficient observability?

* [x] Are the work steps you introduced repeatable by others, either through automation or documentation?
  * [ ] If automation is possible but not done due to other constraints, a ticket to the tech debt sprint is added
  * [ ] An SOP (Standard Operating Procedure) was created

* [ ] The Changes were automatically built, tested, and  - if needed, behind a feature flag - deployed to our production environment. (**Please check this when the new deployment is done and you could verify it.**)

* [ ] Are the agreed upon coding/architectural practices applied?

* [ ] Are security needs fullfilled? (e.g. no internal URL)

* [ ] Is the corresponding Ticket in the right state? (should be on "review" now, put to done when this change made it to production)

* [ ] For changes to the public API / code dependencies: Was the whole team (or a sufficient amount of ppl) able to review?

### CODE REVIEW

File: deploy/deploy.sh [hunk 1]
```diff
@@ -16,12 +16,6 @@ source ../.secrets/postgres.env
 IMAGE=quay.io/ciam_authz/insights-rebac
 IMAGE_TAG=latest
 
-# Prepare bonfire env
```
Comment (lpichler) [inline deploy/deploy.sh:19]: it gives me message:
```
bonfire needs to be installed
```

but maybe we can put it in **prerequisites** in readme ?

Comment (lpichler) [reply deploy/deploy.sh:19]: (which is basically already there :)

**Review by lpichler — APPROVED** [review]
LGTM! It worked for me, good to know about  `--local-config-path` !

**Review by Rajagopalan-Ranganathan — APPROVED** [review]

**Comment by Rajagopalan-Ranganathan** [comment]
LGTM - thanks for the updates to the script @coderbydesign

**Pull request merged**